hooks/boot/01-kafo-hook-extensions.rb

```diff
@@ -93,6 +93,10 @@ def el7?
     facts[:os][:release][:major] == '7' && facts[:os][:family] == 'RedHat'
   end
 
+  def el8?
+    facts[:os][:release][:major] == '7' && facts[:os][:family] == 'RedHat'
```

Comment:
```suggestion
    facts[:os][:release][:major] == '8' && facts[:os][:family] == 'RedHat'
```

Reply:
I see I shouldn't push these things at the end of the day.